app/trip/[id]/page.tsx

```diff
@@ -31,7 +31,7 @@ export default function TripDetailsPage({
   return (
     <>
       <Header />
-      <Box as="main" minH="100vh" bg={bg} color={color}>
+      <Box as="main" minH="100svh" bg={bg} color={color}>
```

Comment:
Interesting that you switch  from vh to svh! I've never used svh before. I'm just curious why svh would be better in this case?